docs/designers-developers/developers/data/data-core-annotations.md

```diff
@@ -8,6 +8,7 @@ Namespace: `core/annotations`.
 
```

Comment:
Seems  like  there's a lot of unrelated changes to markdown files, can we remove these changes from this PR?